source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/skin.html

```
@@ -1,113 +1,564 @@
+					</tr>
+					<tr>
+						<td class='ca_table'>
+							<?tr("Official Containers");?>
+						</td>
+						<td class='ca_stat'>
+							<span data-stat-text='official'></span>
+						</td>
+					</tr>
+					<tr>
+						<td class='ca_table'>
+							<a onclick='event.stopPropagation();showModeration("Repository","<?tr("Repositories");?>");' class='popUpLink'><?tr("Repositories");?></a>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)